gax-java/gax/src/test/java/com/google/api/gax/rpc/ClientContextTest.java

```diff
@@ -1074,4 +1072,46 @@ public void testStreamWatchdogInterval_backportMethodsBehaveCorrectly() {
         ct -> ct.getStreamWatchdogCheckIntervalDuration(),
         ct -> ct.getStreamWatchdogCheckInterval());
   }
+
+  @Test
+  public void testSetApiKey_createsApiCredentials() throws IOException {
```

Comment:
nit: These test methods don't have to be `public` anymore after we migrated to junit 5.

Reply:
ah thanks! done